packages/navigator/src/__tests__/Navigator.test.tsx

```diff
@@ -6,6 +6,8 @@ import Screen from '../Screen'
 
 import { useNavigator } from '../useNavigator'
 
+const NOOP = () => {}
```

Comment:
```suggestion
const noop = () => {}
```

저는 소문자로 쓰긴하는데 ㅎㅎ 혹시 대문자로 쓰시는 이유가 있나요?

Reply:
평소 소문자로 쓰다가 변경하지 않는 상수이니 대문자로 쓰는 게 맞나 싶어서 이번에 대문자로 시도해보았는데, 역시 어색한가보네요. 수정할게요.